counter/src/main/java/edu/nextstep/camp/counter/CounterViewModel.kt

```diff
@@ -0,0 +1,27 @@
+package edu.nextstep.camp.counter
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class CounterViewModel : ViewModel() {
+    private var _count = MutableLiveData(0)
```

Comment:
```suggestion
    private val _count = MutableLiveData(0)
```
mutable livedata는 val로 변경해주세요.